### PR TITLE
Use landing page in flows

### PR DESCRIPTION
**Pull request by lmuntaner**
Main motivation is to use the new page layout for the landing page or II.

<hr/><details><summary>🟡 Some screens were changed</summary></details><details><summary>🔴 Some screens were removed</summary></details>

### CODE REVIEW

**Comment by lmuntaner** [comment]
Clean screenshots:

Screenshot 2024-07-09 at 16 33 12
Screenshot 2024-07-09 at 16 33 52
Screenshot 2024-07-09 at 16 34 08
Screenshot 2024-07-09 at 16 34 42
Screenshot 2024-07-09 at 16 35 15
Screenshot 2024-07-09 at 16 35 25
Screenshot 2024-07-09 at 16 35 32
Screenshot 2024-07-09 at 16 35 52
Screenshot 2024-07-09 at 16 36 00

File: src/frontend/src/components/authenticateBox.ts [hunk 1]
```diff
@@ -569,22 +523,12 @@ export const authnPages = (i18n: I18n, props: AuthnTemplates) => {
...
-    pick: (
-      opts: Parameters<typeof templates.pick>[0],
-      // TODO: remove this parameter once the landing page is ready.
-      landingPageTemplate: boolean = false
-    ) =>
-      // TODO: Use the landing page template always when ready.
-      page(templates.pick(opts), landingPageTemplate),
+      page(templates.useExisting(opts), false),
+    pick: (opts: Parameters<typeof templates.pick>[0]) =>
+      page(templates.pick(opts), true),
```
Comment (frederikrothenberger) [inline src/frontend/src/components/authenticateBox.ts:531]: So the parameter is still there and we maintain both versions of the landing page?

**Review by frederikrothenberger — APPROVED** [review]
LGTM, two points:
* There are no changes to the showcase in this PR. Are all showcase pages still valid / sensible?
* The layout is unfortunately messed up by the banner on the non-official domain (see below). You might want to submit a small fix for that?



Screenshot 2024-07-11 at 10 21 01

File: src/frontend/src/components/authenticateBox.ts (hunk 1 above)
Comment (lmuntaner) [reply src/frontend/src/components/authenticateBox.ts:531]: I'll remove this and clean up the showcase in another PR. Is that ok?

Comment (frederikrothenberger) [reply src/frontend/src/components/authenticateBox.ts:531]: Yes, thanks!

**Pull request merged**